src/test/java/gov/usgs/wma/waterdata/springinit/FileSensingDataSetLoader.java

```diff
@@ -1,24 +1,15 @@
 package gov.usgs.wma.waterdata.springinit;
 
-import java.io.InputStream;
-import java.sql.Date;
-import java.text.SimpleDateFormat;
-import java.time.Instant;
-import java.time.LocalDate;
-import java.time.format.DateTimeFormatter;
-import java.util.AbstractMap;
-import java.util.ArrayList;
-import java.util.List;
-
+import com.github.springtestdbunit.dataset.AbstractDataSetLoader;
 import org.dbunit.dataset.IDataSet;
 import org.dbunit.dataset.ReplacementDataSet;
 import org.dbunit.dataset.csv.CsvURLDataSet;
 import org.dbunit.dataset.xml.FlatXmlDataSetBuilder;
 import org.springframework.core.io.Resource;
 
-import com.github.springtestdbunit.dataset.AbstractDataSetLoader;
-
-import static gov.usgs.wma.waterdata.springinit.BaseIT.getReplacementTokens;
+import java.io.InputStream;
+import java.util.AbstractMap;
+import java.util.List;
```

Comment:
Looks like this file just has imports reorganized. Do not know if this is the "correct" order or if the team coding
standing mentions this. If it is the preferred order, I am fine with. More of a preference, I did not to include
formatting/style change unless the file also has other coding changes.

Reply:
I'm actually trying to avoid "unused imports" and am hitting ctrl-alt-o to "organize imports".   Haven't found a way to automatically delete unused imports.  If there is a preferred order, I'll stop doing this!

Reply:
I think we do have a preferred order so I think restoring the order would be good. Which IDE are you using? If it's IntelliJ, there is a way to effectively import eclipses way of ordering (which is what we use). I don't remember the steps but you can google it and maybe one of the other developers remembers how to do this.

Reply:
ok